lib/requirejs/require.js

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
Non-minified: I don't think it makes much difference for us but we can do it at any moment.

npm: Unlike Almond, RequireJS is not needed for a strictly dev operation in this case (like build). It is required to simply running the editor after cloning the repo, so I would still opt to have it in our code.